habitat/core/registry.py

```diff
@@ -193,35 +206,37 @@ def register_action_space_configuration(
         )
 
     @classmethod
-    def _get_impl(cls, _type, name):
+    def _get_impl(cls, _type: str, name: str) -> Type:
         return cls.mapping[_type].get(name, None)
 
     @classmethod
-    def get_task(cls, name):
+    def get_task(cls, name: str) -> Type["NavigationTask"]:
         return cls._get_impl("task", name)
 
     @classmethod
-    def get_task_action(cls, name):
+    def get_task_action(cls, name: str) -> Type["SimulatorTaskAction"]:
         return cls._get_impl("task_action", name)
 
     @classmethod
-    def get_simulator(cls, name):
+    def get_simulator(cls, name: str) -> Type["HabitatSim"]:
```

Comment:
IIRC, the PyRobot simulator is also in the registry, so this should just be `Type["Simulator"]`

Reply:
Good catch.